.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,49 @@
+name: Bug Report
+description: File a bug report
+title: ":bug: "
+labels: ["Type · Bug", "Status · Unconfirmed"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We appreciate you taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: What issue are you facing?
+      placeholder: Tell us what you see!
```

```suggestion
      placeholder: Tell us what you see!
```
Curious about placeholders here! My understanding is they're generally best avoided (and that's my personal preference) but open to reasoning to the contrary!

---

I copied this from a template and missed this placeholder, I'll remove it! Good catch! 💜